modules/ROOT/pages/_partials/api-gov-concepts.adoc

```diff
@@ -9,21 +9,21 @@ against the rulesets to determine governance conformance.
 
 // end::governance-profiles[]
 
-// tag::governance-profile-status[]
+// tag::gov-active-profiles[]
 
-[[gov-profile-status]]
-governance profile status::
-You can view profile statuses in the API Governance console. Governance profile statuses are based on the percentage of conformant APIs in the profile:
-+
-* Normal: 
-+ 
-More than 70% of APIs are conformant.
-+
-* At Risk:
-+
-Less than 70% of APIs are conformant.
+[[gov-active-profiles]]
+active profiles::
+APIs targeted by active profiles are governed APIs and their conformance information is shown in integrating Anypoint Platform products.
+
+// end::gov-active-profiles[]
+
+// tag::gov-draft-profiles[]
 
-// end::governance-profile-status[]
+[[gov-draft-profiles]]
+draft profiles::
+Use draft profiles to test settings before revealing their targeted APIs' conformance information across Anypoint Platform. APIs targeted by draft profiles are governed APIs but their conformance information is not shown in integrating Anypoint Platform products.
```

Comment:
Maybe switch order of the sentences to maintain parallelism?

Reply:
I added a line "Use active profiles to ..." to make them parallel, since that puts the most important information from a usability perspective first. Please see what you think.

Reply:
That works. I was just thinking that most(?) of the glossary entries are "definition first" then usage.